src/components/Modals/Send/Form.tsx

```diff
@@ -35,21 +35,23 @@ export enum SendFormFields {
   FiatAmount = 'fiatAmount',
   FiatSymbol = 'fiatSymbol',
   AmountFieldError = 'amountFieldError',
-  SendMax = 'sendMax'
+  SendMax = 'sendMax',
+  GasNeed = 'gasNeed'
 }
 
 export type SendInput = {
   [SendFormFields.Address]: string
   [SendFormFields.EnsName]?: string
   [SendFormFields.AccountId]: AccountSpecifier
-  [SendFormFields.AmountFieldError]: string | [string, { asset: string }]
+  [SendFormFields.AmountFieldError]: string | [string, { asset: string; amount: string }]
   [SendFormFields.Asset]: Asset
   [SendFormFields.FeeType]: chainAdapters.FeeDataKey
   [SendFormFields.EstimatedFees]: chainAdapters.FeeDataEstimate<ChainTypes>
   [SendFormFields.CryptoAmount]: string
   [SendFormFields.CryptoSymbol]: string
   [SendFormFields.FiatAmount]: string
   [SendFormFields.FiatSymbol]: string
+  [SendFormFields.GasNeed]: string
```

Comment:
Typo: `GasNeeded`.

Reply:
@gomesalexandre What would be the best user experience for this feature?

Reply:
Sorry for the late reply, looks like I missed this one whilst in the middle of switching mail providers.

The spec will still need to be defined by design, but since it's already bountied, the current spec should work for this one. No need to show the amount of additional gas or anything on this one, just get in errored button state and don't let user continue the trade flow. Feel free to ping me on discord if the ACC isn't clear!